services/provider/proto/provider.proto

```diff
@@ -118,13 +118,30 @@ message StorageConfigResponse{
     SystemAttributes systemAttributes = 3;
 }
 
+message RbdDriverRequirements {
+  bool generateOmapInfo = 1;
```

Comment:
You are surfacing requirements so it does not make sense to have something named as a verb (generate)
```suggestion
  bool omapInfo = 1;
```